images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -149,6 +149,12 @@ function Get-GradleVersion {
     return "Gradle $gradleVersion"
 }
 
+function Get-SbtVersion {
+    ($(sbt -version) | Out-String) -match "sbt script version: (?<version>\d+\.\d+\.\d+)" | Out-Null
```

Comment:
nitpick : `(sbt -version) -match "sbt script version:" -replace "script version: "`

Reply:
It's way more readable, thanks! 🤝